charts/vault-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: vault-operator
-version: 1.20.0-dev.1
+version: 1.20.0-dev.2
```

Comment:
I think the `version` should be 1.20.1 and the `appVersion` 1.20.0, right, @sagikazarmark?

Reply:
@akijakya , thank you so much for the comment and feedback. I have updated the versions. Thanks

Reply:
We no longer need to update versions in PRs, so it doesn't really matter anymore.